src/client/css/partials/breaches.css

```diff
@@ -136,11 +136,19 @@
 
 .breaches-table header > :nth-child(3),
 .breaches-table summary > :nth-child(3) {
+  flex: 1 0 80px;
+  display: flex;
+  justify-content: flex-end;
+}
+
+.breaches-table header > :nth-child(4),
+.breaches-table summary > :nth-child(4) {
   flex-basis: 100px;
 }
 
 .breaches-table summary > :nth-child(2),
-.breaches-table summary > :nth-child(3) {
+.breaches-table summary > :nth-child(3),
```

Comment:
Haha, I think we have different definitions of giant :sweat_smile: This is what it looks like for me:

![image](https://user-images.githubusercontent.com/4251/217518621-eb0cad91-71ac-4f67-9d51-8ce4cfd37d02.png)

I see a height of 24px for the badge in the UX design:

![image](https://user-images.githubusercontent.com/4251/217519333-a980626f-7841-4a3f-809d-8dcaa145bc64.png)

That said, when it comes to the issue @flozia pointed out, the problem is that `summary` has a fixed height, to ensure that elements that get flex-wrapped are hidden from view, rather than taking up another row. But the badge is higher than the rest of the text in a row, because of its vertical padding. I can increase the `summary` height or decrease its padding to adjust it for the badge height, but then that breaks down as soon as the higher elements disappear.

Without using media queries I'm not sure if I have a better solution than just making the badge not be taller than the rest of the text in the summary element, so I did that in 5476b557.

Reply:
heh – yeah, I guess "giant" was an exaggeration!  

My point was the dynamic badge size brings its font-size to 16px when the designs suggest 11px.  We got UX approval for dynamic font size on the other items, but I suspect the badge is a different case: the bolded larger size with color takes my focus over the company name.

Looks like this is addressed via https://github.com/mozilla/blurts-server/commit/5476b55792837a85f628fc1005f9da71afa34b64 👍 

@Vinnl Should we use `rem` to stay consistent with the rest of the site?

Reply:
Oops, yes we should: https://github.com/mozilla/blurts-server/pull/2822